modules/library/coverage/src/main/java/org/geotools/coverage/processing/OperationJAI.java

```diff
@@ -16,12 +16,20 @@
  */
 package org.geotools.coverage.processing;
 
+import it.geosolutions.jaiext.JAIExt;
```

Comment:
The file has been modified, its last copyright year should be pushed to 2015. The same is valid for any other pre-existing modified file, I won't repeat this comment over and over, please fix them all